### PR TITLE
Add Udacity

### PR DESCRIPTION
**Pull request by thedeerchild**
This is a modified version of the [Contributing Guidelines](/CONTRIBUTING.md).

This pull request adheres to the repository's [Code of Conduct](/CODE_OF_CONDUCT.md)

- [x] This PR contains housekeeping (URL edits, copy changes etc)
- [x] You know your alphabet - company is listed in alphabetical order in the README
- [x] A company profile is included - Not essential but it really helps!
- [x] The company directly hires employees - This may sounds odd but while awesome, this list is not for coding bootcamps, "teaching" positions a network (Teacher positions employed directly, such as Treehouse, are allowed), or networking sites for listing a CV looking for gigs
- [x] The company added hires remote employees, or positions are available to remote workers and are clearly illustrated as such
- [x] A company-profile has been included for each addition to the list
- [x] I am an employee of the company mentioned and confirm all details are correct
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details any restrictions to applicants based on geography
- [x] __How to apply__ details the best approach for new applications, link to open positions, and any other help available for job hunters
- [x] Any missing parts from the profile, please use \<Unknown\> to allow for future completion

### CODE REVIEW

File: company-profiles/udacity.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+We believe in working however suits you best; not in measuring productivity by how often you come into the office. Whether you prefer our main office in Mountain View, our co-working space in San Francisco, a home office on the East Coast or a beach cafe in some tropical paradise, we'll make sure you're able to productively collaborate and communicate with the rest of your team via Slack, Hangouts and GitHub.
+
+## Region
+
+Worldwide
+
+## Company technologies
+
+Much like *where* we work, we belive *how* we work should be dictated by productivity and fit for the task at hand. Our team uses a number of technologies (React/GraphQL, Ruby on Rails, Go, Scala, Python, Angular, Java and Objective-C to name a few), but you're welcome to use whatever language/framework/dev environment will let you do your best work.
```
Comment (gbraad) [inline company-profiles/udacity.md]: believe

**Comment by gbraad** [comment]
Small typo; I marked this in the change itself. (s/belive/believe)

**Comment by thedeerchild** [comment]
Thanks for catching that @gbraad

**Comment by dougaitken** [comment]
Thanks @thedeerchild @gbraad!

**Pull request merged**